Userland/Utilities/nc.cpp

```diff
@@ -27,6 +27,24 @@
 // output to stderr to allow for commands like:
 //
 // nc -l someport > out.file
+//
+// Below posix spec for basic socket system interface
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/V2_chap02.html
+//
+// Below reference implementation was considered to come up default bounds
+// for SO_RCVBUF
+// https://man7.org/linux/man-pages/man7/socket.7.html
+static constexpr std::size_t max_tcp_rcv_buff_upper = 212992;
+static constexpr std::size_t max_tcp_rcv_buff_lower = 256;
```

Comment:
Please don't randomly abbreviate identifiers. We don't have a limit on horizontal length.
Please use `size_t` instead of `std::size_t`.